content/docs/validators/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Validators"
+linkTitle: "Validators"
+weight: 50
+description: >
+    Instructions for setting up and operating a validator node.
+---
+
+A node on the Autonity can register to become a [Validator](/concepts/validator/), making it eligible for selection into the [consensus committee](/concepts/consensus/committee/) to earn rewards for proposing and finalizing blocks.
```

Comment:
```suggestion
A node on the Autonity can be registered to become a [Validator](/concepts/validator/), making it eligible for selection into the [consensus committee](/concepts/consensus/committee/) to earn rewards for proposing and finalizing blocks.
```

Reply:
Edit good, but incomplete:

>A node on the Autonity can be registered to become a

Change to:

>A node on an Autonity network can be registered to become a

Reply:
updated